### PR TITLE
Added validate-service status after cloud service is deployed , to mak…

### PR DESCRIPTION
**Pull request by jagany**
…e sure task verify the deployment, waits until the instances deployed by service are in ready State

### CODE REVIEW

File: Tasks/AzureCloudPowerShellDeployment/task.json [hunk 1]
```diff
@@ -1,224 +1,229 @@
...
-        "_0couldnotbeparsedintopartsforregisteringdiagnosticsextensions": "'{0}' could not be parsed into parts for registering diagnostics extensions.",
-        "Nofileswerefoundtodeploywithsearchpattern0": "No files were found to deploy with search pattern {0}",
-        "Foundmorethanonefiletodeploywithsearchpattern0Therecanbeonlyone": "Found more than one file to deploy with search pattern {0}. There can be only one.",
-        "Storagekeysaredefinedininvalidformat": "Storage keys are defined in invalid format.",
-        "Unabletofind0usingprovidedsubscription": "Unable to find {0} using provided subscription",
-        "Currentversionofazurepowershelldontsupportexternalstorageaccountforconfiguringdiagnostics": "Current version of azure powershell does not support external storage account for configuring diagnostics.",
-        "Customcertificatesaredefinedininvalidformat": "Custom certificates are defined in invalid format.",
-        "Addinganyconfiguredcustomcertificates": "Adding any configured custom certificates."
-    }
+	"id": "2CA8FE15-42EA-4B26-80F1-E0738EC17E89",
```
[the remark this answers is not in the document]
Comment (Ajay-MS) [reply Tasks/AzureCloudPowerShellDeployment/task.json]: It seems indentation format for the complete file has been changed. Please take original file format and do necessary changes only.

File: Tasks/AzureCloudPowerShellDeployment/Publish-AzureCloudDeployment.ps1 [hunk 2]
```diff
@@ -143,6 +143,10 @@ try{
 
 
 } finally {
+	        Write-Host "Validate-ServiceStatus -ServiceName $ServiceName -Slot $Slot"
```
[the remark this answers is not in the document]
Comment (jagany) [reply Tasks/AzureCloudPowerShellDeployment/Publish-AzureCloudDeployment.ps1]: I decided to remove as we already have the messages coming from the Validate-SreviceStatus method

File: Tasks/AzureCloudPowerShellDeployment/Utility.ps1 [hunk 3]
```diff
@@ -254,4 +254,81 @@ function Add-CustomCertificates($serviceName, $customCertificatesMap)
             Write-Verbose "Deleted the temp file $tmpFile.FullName"
         }
     }
+}
+
+function Validate-ServiceStatus ($ServiceName, $Slot)
+{
+		$RoleInstances=$null;
```
[the remark this answers is not in the document]
Comment (jagany) [reply Tasks/AzureCloudPowerShellDeployment/Utility.ps1]: Updated

**Comment by jagany** [comment]
Created PR for the same from different fork, closing this PR

**Pull request closed without merging**